src/components/TocItem/TocItem.tsx

```diff
@@ -55,6 +55,20 @@ class TocItem extends React.Component<TocItemProps> {
             rel: isExternal ? 'noopener noreferrer' : undefined,
         };
 
+        if (active) {
```

Comment:
there is no need to copy the whole element just to add a label, you can do something like

```
        return (
            <a
                {...linkAttributes}
                className={b('link')}
                onClick={expandable && href ? this.handleClick : undefined}
                data-router-shallow
                aria-current={active ? 'true' : undefined}
            >
                {content}
            </a>
        );
```